### PR TITLE
Revise serverside

### PR DESCRIPTION
**Pull request by isc-rsingh**
server side revisions of documentation

### CODE REVIEW

File: docs/ServerSide.md
Comment (gjsjohnmurray) [inline docs/ServerSide.md:73]: I think the correct format for this is

```suggestion
			"uri": "isfs://iris184:user/"
```

File: docs/ServerSide.md
Comment (gjsjohnmurray) [inline docs/ServerSide.md:77]:
```suggestion
			"uri": "isfs-readonly://iris184:%sys/"
```

File: docs/ServerSide.md
Comment (gjsjohnmurray) [inline docs/ServerSide.md:53]:
```suggestion
			"uri": "isfs://iris184:user/"
```

File: docs/ServerSide.md
Comment (gjsjohnmurray) [inline docs/ServerSide.md:63]:
```suggestion
   - The value following the next `:` specifies the namespace (lowercase).
```

File: docs/ServerSide.md
Comment (gjsjohnmurray) [inline docs/ServerSide.md:59]:
```suggestion
- The `name` property provides a name for this server-side folder.
```

File: docs/ServerSide.md
Comment (gjsjohnmurray) [inline docs/ServerSide.md:45]:
```suggestion
Note that the ObjectScript button is not visible in the Activity Bar, because the files listed in the Explorer view are all on the server, so it is not needed for this configuration.
```

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]
A few changes needed. 

The server-side stuff will benefit from a further revision to reflect the UI that arrived in Server Manager 2.0, but I suggest we merge and publish the existing updates as soon as my changes have been incorporated.

**Comment by gjsjohnmurray** [comment]
@isc-rsingh whatever happened to review before merge?

**Comment by isc-rsingh** [comment]
Haha @gjsjohnmurray I told Jan you would probably say something! We prioritized learning the full workflow over waiting for the approval, but that certainly won't be the norm.

**Pull request merged**
